src/Microsoft.OData.Core/ODataMetadataJsonOutputContext.cs

```diff
@@ -55,7 +55,7 @@ internal sealed class ODataMetadataJsonOutputContext : ODataOutputContext
                 }
                 else
                 {
-                    this.asynchronousOutputStream = new AsyncBufferedStream(this.messageOutputStream);
+                    this.asynchronousOutputStream = new BufferedStream(this.messageOutputStream, 84000);

```

Comment:
It would be good to go ahead and make `84000` configurable through the constructor

Reply:
There seem to be other places where this is the case too

Reply:
Do we really need this configurable? For example. the old implementation of the `DataBuffer` also used a fixed size of 79kb, and it wasn't configurable.

Reply:
It's definitely not a need, which is why I approved anyway. But I don't think it is that large of a change to address if you have the time.

Reply:
I think saving the number in constant would be better for maintainability.

Reply:
Done